Newtonsoft.Json dependency warning

I'm using newest Hangfire + Hangfire/AspNetCore packages and on build I get strange warning:

```warning NU1603: Hangfire.Core 1.6.15 depends on Newtonsoft.Json (>= 5.0.0) but Newtonsoft.Json 5.0.0 was not found. An approximate best match of Newtonsoft.Json 5.0.1 was resolved.```

I have Newtonsoft.Json 10.0.3, so why the warning?

![image](https://user-images.githubusercontent.com/1478006/29486266-03093cbc-84e2-11e7-9c2b-b6ce86c6c0f1.png)

--- a/nuspecs/Hangfire.Core.nuspec
+++ b/nuspecs/Hangfire.Core.nuspec
@@ -130,7 +130,7 @@
     <dependencies>
       <group targetFramework="net45">
         <dependency id="Owin" version="1.0" />
-        <dependency id="Newtonsoft.Json" version="5.0.0" />
+        <dependency id="Newtonsoft.Json" version="5.0.1" />
       </group>
       <group targetFramework="netstandard1.3">
         <dependency id="Microsoft.NETCore.Portable.Compatibility" version="1.0.1" />

